### DISCUSSION TITLE
Open different card with tap_action

### DISCUSSION THREAD

**Discussion by dejongac** [marcokreeft87/room-card#27] [Q&A]
Hi Maybe the wrong place, but i would like to open a different card when i tap a icon.
When i now tap the climate icon i get the more_info dialog, instead of the thermostat card or 
mushroom-climate-card. Which is more user friendly.

Thanks

Albert

**Comment by marcokreeft87** [comment]
What you could do is make a template input boolean and toggle that with a tap of an icon and make the card you wanna show conditional on that input_boolean. But this is nothing room card can or will do. So you have to make that yourself with templating

**Reply by dejongac** [reply]
Thanks Mark for the response.

I love how the room card displays the information, so i was using it to create a wall tablet. But when someone wants to change temperature they need to have something fancy otherwise the acceptance factor is low. I will dive into your suggestions.

**Reply by marcokreeft87** [reply]
You can add cards to the room card. They will displayed below the entities. There is an example on github
